src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+.product-card {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 0 16px 16px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 32px auto 40px;
+  }
+
+  &__name {
+    display: inline-block;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $text-main-color;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $text-additional-colour;
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__number {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $text-main-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    line-height: 18px;
+    margin-bottom: 16px;
+  }
+
+  &-word {
+    font-weight: 400;
+    font-size: 12px;
+    color: $text-additional-colour;
+  }
+
+  &-value {
+    font-weight: 700;
+    font-size: 16px;
+    color: $text-main-color;
+  }
+
+  &__button {
+    display: block;
+    width: 100%;
+    height: 40px;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 40px;
+    text-transform: uppercase;
+    text-decoration: none;
+    text-align: center;
+    color: $background-color;
+    background-color: $accent-colour;
+    border-radius: 5px;
+    border: 1px solid transparent;
+    transition: 0.3s;
+    cursor: pointer;
+
+    &:hover {
+      background-color: $background-color;
+      border: 1px solid $accent-colour;
+      color: $accent-colour;
+      flex-direction: column;
+    }
+  }
+}
+
+.content__product-card {
```

Comment:
This is selector of another BEM block, so it should be written in a separate file.